apps/server/src/modules/tldraw/controller/api-test/tldraw.ws.api.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { WsAdapter } from '@nestjs/platform-ws';
+import { Test } from '@nestjs/testing';
+import WebSocket from 'ws';
+import { TextEncoder } from 'util';
+import { INestApplication } from '@nestjs/common';
+import { CoreModule } from '@src/core';
+import { ConfigModule } from '@nestjs/config';
+import { createConfigModuleOptions } from '@src/config';
+import { config } from '../../config';
+import { TldrawWsService } from '../../service';
+import { TldrawBoardRepo } from '../../repo';
+import { TldrawWs } from '../tldraw.ws';
+import { TestHelper } from '../../helper/test-helper';
+
+describe('WebSocketController (WsAdapter)', () => {
```

Comment:
Yes, that is good point. We thought about it, but there is an issue with mocking tldraw clients set of messages so that the server recognizes it as valid y-doc data.
In future we surely will try to make full flow test